code/modules/research/xenoarchaeology/tools/tools_depthscanner.dm

```diff
@@ -153,11 +153,17 @@
 	var/adv = FALSE
 
 /obj/item/device/xenoarch_scanner/adv
-	name = "advanced xenoarch digsite locator"
-	icon_state = "forensic1-old" //placeholder
-	desc = "A scanner that scans the surrounding area for potential xenoarch digsites, highlighting them temporarily in a colour associated with their responsive reagent. REquires mesons for optimal use."
+	name = "Advanced Xenoarchaeological digsite locator"
+	icon_state = "unknown"
+	desc = "A scanner that scans the surrounding area for potential xenoarch digsites, highlighting them temporarily in a colour associated with their responsive reagent. Requires mesons for optimal use."
 	adv = TRUE
 
+/obj/item/device/xenoarch_scanner/adv/examine(mob/user)
+	..()
+	to_chat(user, "<span class = 'notice'>It has a list of colour codes:</span>")
+	for(var/i in color_from_find_reagent)
+		to_chat(user, "[i] = <span style = 'color:[color_from_find_reagent[i]];'>this colour</span>")
```

Comment:
color

Reply:
is the string going to dick something or are we just being stylistic

Reply:
The CSS uses the american "color" but out of habit I put colour instead of color for the regular text.